haystack/backends/solr_backend.py

```diff
@@ -738,3 +753,30 @@ def run_mlt(self, **kwargs):
 class SolrEngine(BaseEngine):
     backend = SolrSearchBackend
     query = SolrSearchQuery
+
+
+class SolrSchemaAdmin(object):
```

Comment:
Right now, a change in `search-indexes.py` doesn't update the Solr schema. I've made a correction to this class on [my fork of this repo](https://github.com/ahoho/django-haystack/commit/b9f58e0e4e7e560c14b9958befc1fb797f0fb784), which also requires a minor change in `build_solr_schema.py`